docs/source/models/cause.rst

```diff
@@ -558,6 +558,95 @@ below.
 Remission Rates
 ^^^^^^^^^^^^^^^
 
+Definition
+""""""""""
+Remission is a measure of cases that recover from a with-condition state, given 
+a specified population and time period. Just as with incidence, the numerator is 
+given by the count of recovered (or *remitted*) cases, and the denominator is 
+the cumulative person-time [link to person-time definition in incidence 
+section] during which cases are able to go into remission:
```

Comment:
@beatrixkh Here are instructions for referencing an arbitrary location in a document: http://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#cross-referencing-arbitrary-locations

Reply:
thank you!